src/pages/Login/Login.styled.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components/macro'
+import { theme } from '../../ui'
```

Comment:
Use os paths do TS para importação de arquivos, nesse caso ficaria:

```suggestion
import { theme } from 'ui'
```